### PR TITLE
add sample for amp-3d-gltf

### PR DESCRIPTION
**Pull request by mixtur**
Addresses https://github.com/ampproject/amp-by-example/issues/1297

### CODE REVIEW

File: src/20_Components/amp-3d-gltf.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<!--
+  ## Introduction
+
+  The amp-3d-gltf component displays 3D models that are in glTF format.
```
Comment (sebastianbenz) [inline src/20_Components/amp-3d-gltf.html]: Can you please link `amp-3d-gltf` to the official docs.

File: src/20_Components/amp-3d-gltf.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  <!--
+    Import the `amp-3d-gltf` component in the header.
+  -->
+  <script async custom-element="amp-3d-gltf" src="https://cdn.ampproject.org/v0/amp-3d-gltf-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+
+  <!-- ## Basic Usage -->
+  <!-- **Example 1: Embed 3d model via the `src` attribute** -->
```
Comment (sebastianbenz) [inline src/20_Components/amp-3d-gltf.html]: For consistency reasons reasons, can you please remove `**` and `Example 1`.

File: src/20_Components/amp-3d-gltf.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  <amp-3d-gltf layout="fixed"
+               width="320"
+               height="240"
+               src="https://webgears-3d.github.io/3d-gltf-static/m/DamagedHelmet.glb"></amp-3d-gltf>
```
Comment (sebastianbenz) [inline src/20_Components/amp-3d-gltf.html]: Would you mind hosting the glb file on ampbyexample? Simply put it into the static dir.

File: src/20_Components/amp-3d-gltf.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  <!-- ## Attributes -->
+  <!--
+    This section covers how you can customize the viewer for your needs.
+    * src - A required attribute that specifies the URL to the glTF file.
```
Comment (sebastianbenz) [inline src/20_Components/amp-3d-gltf.html]: Can you please surround the attribute names with ` for code formatting.

**Comment by sebastianbenz** [comment]
The AMP validator does not yet seem to support all features of amp-3d-gltf. When is it going to be updated?

**Comment by mixtur** [comment]
I am waiting for some response in this issue https://github.com/ampproject/amphtml/issues/15793

File: src/20_Components/amp-3d-gltf.html (hunk 3 above)
Comment (mixtur) [reply src/20_Components/amp-3d-gltf.html]: I don't mind but there are two problems with that.
- `src` must resolve to https, therefore I cannot check if it works locally. dev-server is not https.
- `/` - based urls in src seems to be broken in amp-3d-gltf anyway. I sent a [PR](https://github.com/ampproject/amphtml/pull/15832) fixing it. Should probably wait until it is merged.

File: src/20_Components/amp-3d-gltf.html (hunk 1 above)
Comment (mixtur) [reply src/20_Components/amp-3d-gltf.html]: done

File: src/20_Components/amp-3d-gltf.html (hunk 2 above)
Comment (mixtur) [reply src/20_Components/amp-3d-gltf.html]: done

File: src/20_Components/amp-3d-gltf.html (hunk 4 above)
Comment (mixtur) [reply src/20_Components/amp-3d-gltf.html]: done

**Comment by mixtur** [comment]
Validator is fixed, and relative paths should work in amphtml's master. I don't know when those changes will be visible from this project though.

**Comment by mixtur** [comment]
Thanks to recent release, it seems to work on my machine. Though I had to add `cors: true` in your serve task (didn't committed that)

**Comment by lannka** [comment]
> Though I had to add cors: true in serve task in gulpfile (didn't committed that)
the live example is currently broken due to the CORS issue.

@sebastianbenz would you help to enable CORS ? I created an issue to track #1446

**Pull request merged**